policy/modules/contrib/afterburn.te

```diff
@@ -37,6 +40,18 @@ domain_use_interactive_fds(afterburn_t)
 
 files_read_etc_files(afterburn_t)
 
+manage_dirs_pattern(afterburn_t, afterburn_tmp_t, afterburn_tmp_t)
+manage_files_pattern(afterburn_t, afterburn_tmp_t, afterburn_tmp_t)
+files_tmp_filetrans(afterburn_t, afterburn_tmp_t, { file dir })
```

Comment:
I saw only a directory reported in /tmp, so a plain file can also be created?

Reply:
This section should be before kernel_, t.w. with the related mounton permission.

Reply:
It's actually only `dir`. I added `file` out of an abundance of caution, but I can remove it and re-test.

Reply:
Now I am actually surprised that no rules for files are needed.